tests/Microsoft.Bot.Streaming.Tests/WebSocketTransportTests.cs

```diff
@@ -89,6 +89,27 @@ public async Task WebSocketClient_ConnectThrowsIfUrlIsBad()
             Assert.IsType<UriFormatException>(result);
         }
 
+        [Fact]
+        public async Task WebSocketClient_ConnectAsyncExThrowsIfUrlIsBad()
+        {
+            Exception result = null;
+            WebSocketClient reader = null;
+            try
+            {
+                var webSocketClient = new WebSocketClient("fakeurl");
+                reader = webSocketClient;
+
+                await reader.ConnectAsyncEx(null, CancellationToken.None);
```

Comment:
You could do something like Assert.Throws<UriFormatException>( async () => await reader.ConnectAsyncEx(null, CancellationToken.None))